cmd/setup-gh.go

```diff
@@ -203,6 +203,6 @@ func GetAzSubscriptionId(subIds []string) string {
 }
 
 func init() {
-	rootCmd.AddCommand(newSetUpCmd())
-
+	ctx := context.Background()
```

Comment:
there's another way to get the context that's better than setting it like this. ctx should be passed from the cli framework.

https://github.com/Azure/aks-spin-plugin/blob/7d86429bbf2cd5ce17df57ed676063b5834be96f/cmd/init.go#L20